Code/src/main/java/nl/utwente/viskell/ghcj/GhciEvaluator.java

```diff
@@ -20,7 +20,7 @@ public GhciEvaluator() throws HaskellException {
 
     @Override
     protected List<String> getModules() {
-        return ImmutableList.of("Data.List", "Data.Maybe", "Data.Either", "Data.Monoid", "Control.Applicative");
+        return ImmutableList.of("Data.List", "Data.Maybe", "Data.Either", "Data.Monoid", "Control.Applicative", "Test.QuickCheck");
```

Comment:
Would it be doable to catch the ghci error messages on startup and show that to the user?